apps/nova/src/lib/room.test.ts

```diff
@@ -9,6 +9,7 @@ describe(nameof(asOtherMember), () => {
     id: 1,
     email: faker.internet.email(),
     name: faker.person.fullName(),
+    bio: "",
```

Comment:
Let's add some value to verify that is working as expected. Also add for the `other` member.